### PR TITLE
Instructions for docker and introduce google-cloud-bigtable-emulator

### PR DESCRIPTION
**Pull request by rahulKQL**

### CODE REVIEW

**Comment by rahulKQL** [comment]
@romlogic Please have a look

File: bigtable-test/bigtable-emulator-maven-plugin/README.md [hunk 1]
```diff
@@ -9,6 +9,7 @@ It provides 3 maven goals:
 * stop: stops the emulator
 * run: like start, but runs the emulator in foreground
 
+_Note: You may checkout [Google-cloud-Bigtable-emulator](#google-cloud-bigtable-emulator)._
```
Comment (romlogic) [inline bigtable-test/bigtable-emulator-maven-plugin/README.md]: check out in this case is two words if you are using it as verb.

File: bigtable-test/bigtable-emulator-maven-plugin/README.md [hunk 2]
```diff
@@ -66,4 +67,47 @@ Usage:
       Connection connection = BigtableConfiguration.connect("fakeproject", "fakeinstace");
```
Comment (romlogic) [inline bigtable-test/bigtable-emulator-maven-plugin/README.md]: "fakeproject", "fakeinstace" spelling error; should be "fakeinstance"

File: bigtable-test/bigtable-emulator-maven-plugin/README.md [hunk 3]
```diff
@@ -66,4 +67,47 @@ Usage:
...
    ```
 - If `maven.test.skip` is set to true the emulator will not start. The emulator can also be controlled directly 
-  by setting the boolean property `bigtable.emulator.skip`.
\ No newline at end of file
+  by setting the boolean property `bigtable.emulator.skip`.
+
+- While using Docker, You can use container's `host` and `port` details for emulator:
+  ```java
+  GenericContainer container = // Container Contorller
```
Comment (romlogic) [inline bigtable-test/bigtable-emulator-maven-plugin/README.md]: controller spelled incorrect

File: bigtable-test/bigtable-emulator-maven-plugin/README.md [hunk 4]
```diff
@@ -66,4 +67,47 @@ Usage:
       Connection connection = BigtableConfiguration.connect("fakeproject", "fakeinstace");
    ```
 - If `maven.test.skip` is set to true the emulator will not start. The emulator can also be controlled directly 
-  by setting the boolean property `bigtable.emulator.skip`.
\ No newline at end of file
+  by setting the boolean property `bigtable.emulator.skip`.
+
+- While using Docker, You can use container's `host` and `port` details for emulator:
```
Comment (romlogic) [inline bigtable-test/bigtable-emulator-maven-plugin/README.md]: While..., you can use the container's...(You should not be capilatlized and add the)

File: bigtable-test/bigtable-emulator-maven-plugin/README.md [hunk 5]
```diff
@@ -66,4 +67,47 @@ Usage:
...
+
+  Configuration config = BigtableConfiguration.configure("fakeproject", "fakeinstance");
+  config.set(container.getContainerIpAddress() + ":" + container.getMappedPort(TEST_HTTP_PORT));
+  Connection connection = BigtableConfiguration.connect(config);
+  ```
+
+
+## [Google-cloud-Bigtable-emulator](https://github.com/googleapis/google-cloud-java/tree/master/google-cloud-testing/google-cloud-bigtable-emulator)
+
+  This is the latest bigtable emulator, It includes below features over `bigtable-emulator-maven-plugin`:
```
Comment (romlogic) [inline bigtable-test/bigtable-emulator-maven-plugin/README.md]: use a semi colon, or a full stop. If semi colon, do not start with capital "I". If you are using a full stop, then you can start with a capital letter.

File: bigtable-test/bigtable-emulator-maven-plugin/README.md [hunk 6]
```diff
@@ -66,4 +67,47 @@ Usage:
...
+  - Supports JUnit Rules
+  - Allows for direct programmatic control
+  - Bundles the emulator binary
+  - You can use a direct emulator wrapper as (i.e. [Emulator.java](https://github.com/googleapis/google-cloud-java/blob/master/google-cloud-testing/google-cloud-bigtable-emulator/src/main/java/com/google/cloud/bigtable/emulator/v2/Emulator.java))
+
+  Example usage with Cloud-Bigtable-Client:
```
Comment (romlogic) [inline bigtable-test/bigtable-emulator-maven-plugin/README.md]: Example of ...

**Comment by rahulKQL** [comment]
Thanks, @romlogic for review

**Pull request closed without merging**